src/coreclr/jit/emitarm64.h

```diff
@@ -682,11 +735,22 @@ inline static bool isFloatReg(regNumber reg)
     return isVectorRegister(reg);
 }
 
+inline static bool isSveRegister(regNumber reg)
+{
+    // TODO-SVE: Fix once we add Z registers
+    return (reg >= REG_FP_FIRST && reg <= REG_FP_LAST);
```

Comment:
"z" registers are alias of "v" registers so this will be exactly similar to `isVectorRegister()`, so was not sure if we need a separate "isSveRegister()`.

Reply:
Even if we add `REG_Z0`, `isSveRegister()` will still be the same code as `isVectorRegister()`.
Removed.